cmake/loki_transform.cmake

```diff
@@ -291,26 +291,6 @@ function( loki_transform_target )
             list( APPEND _TRANSFORM_OPTIONS CPP )
         endif()
 
-        if( _PAR_T_INLINE_MEMBERS )
```

Comment:
With these removed, we should now also remove the corresponding arguments from the CMake function - that will make sure that any options that have been specified are not silently ignored but do actually raise an error in CMake directly. For that, ll. 226-231 need to look like this:

```cmake
    set( options NO_PLAN_SOURCEDIR COPY_UNMODIFIED CPP CPP_PLAN )
    set( single_value_args TARGET COMMAND MODE FRONTEND CONFIG PLAN )
    set( multi_value_args SOURCES HEADERS DEFINITIONS INCLUDES )
```

Please also update the docstring to match the reduced interface.